routes/calls/calls.js

```diff
@@ -17,9 +17,9 @@ exports.searchCalls = function(req, res){
 	if(common.isAuthenticated(req, res)) {
 		var query = req.query;
 		if(req.query.today === 'true'){
```

Comment:
@tremainebuchanan you need some documentation here... be careful how you use the entity comparison.  If you are expecting boolean try and use boolean `true` and the `==` comparator.